pkgs/by-name/gi/githooks/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  buildGo122Module,
+  fetchFromGitHub,
+  git,
+  testers,
+  makeWrapper,
+}:
+buildGo122Module rec {
+  pname = "githooks";
+
+  version = "3.0.1";
+
+  src = fetchFromGitHub {
+    owner = "gabyx";
+    repo = "githooks";
+    rev = "v${version}";
+    hash = "sha256-qv0nl3EEYVo/s79r+yK3ZQCGPXM2bzGdWatPY24aOZg=";
+  };
+
+  modRoot = "./githooks";
+  vendorHash = "sha256-ZcDD4Z/thtyCvXg6GzzKC/FSbh700QEaqXU8FaZaZc4=";
+  nativeBuildInputs = [makeWrapper];
+  buildInputs = [git];
+
+  ldflags = [
+    "-s" # Disable symbole table.
+    "-w" # Disable DWARF generation.
+  ];
+
+  tags = ["package_manager_enabled"];
+
+  doCheck = false;
+
+  postConfigure = ''
```

Comment:
Is this correct?

Reply:
Yes we need to configure some build files before building.